code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -461,3 +461,17 @@
 	reagent_state = LIQUID
 	color = "#29262b"
 	taste_description = "burnt dirt"
+
+/datum/reagent/tar_compound
+	name = "Sticky tar"
+	id = "sticky_tar"
+	description = "A sticky compound that creates tar on contact with surfaces."
+	reagent_state = LIQUID
+	color = "#4B4B4B"
+	harmless = FALSE
+	taste_description = "processed sludge"
+
+/datum/reagent/tar_compound/reaction_turf(turf/simulated/loc, volume)
+	if(volume >= 1 && istype(loc, /turf/simulated))
```

Comment:
```suggestion
	if(volume >= 1 && istype(loc))
```
since `loc` is typecasted to simulated turf anyway, you don't actually need to specify the path to check, it knows implicitely.